pkgs/development/python-modules/interface_meta/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "interface_meta";
+  version = "1.3.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "matthewwardrop";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DSL1cS0sz1epIZaLsISwnVnkrBdshzRCWGMsZXkI8Gc=";
+  };
+
+  preBuild = ''
+    substituteInPlace pyproject.toml  \
+      --replace 'version = "0.0.0"' 'version = "${version}"'  \
+      --replace "enable = true" "enable = false"
```

Comment:
This could easily substitute something wrong. Is there more on the line? Otherwise we should think about a patch.

Reply:
Removed this line, didn't have any effect.